WowPacketParserModule.V4_3_4_15595/Parsers/GuildHandler.cs

```diff
@@ -477,15 +480,15 @@ public static void HandleGuildBankList434(Packet packet)
             {
                 for (var j = 0; j < enchants[i]; ++j)
                 {
-                    packet.ReadUInt32("Enchantment Slot Id?", i, j);
-                    packet.ReadUInt32("Enchantment Id?", i, j);
+                    packet.ReadUInt32("GemSlot", i, j); // 0 - 2
+                    packet.ReadUInt32("GemEnchantmentID", i, j);
                 }
-                packet.ReadUInt32("Unk UInt32 1", i); // Only seen 0
-                packet.ReadUInt32("Unk UInt32 2", i); // Only seen 0
-                packet.ReadUInt32("Unk UInt32 3", i); // Only seen 0
+                packet.ReadUInt32("PermanentEnchantmentID", i);
+                packet.ReadUInt32("ReforgeEnchantmentID", i);
+                packet.ReadUInt32("BonusEnchantmentID", i);
                 packet.ReadUInt32("Stack Count", i);
                 packet.ReadUInt32("Slot Id", i);
-                packet.ReadUInt32E<UnknownFlags>("Unk mask", i);
+                packet.ReadUInt32E<UnknownFlags>("ItemFlags", i);
```

Comment:
are these flags meant? https://gitlab.com/trinitycore/TrinityCore_434/blob/4.3.4/src/server/game/Entities/Item/ItemTemplate.h#L154

Reply:
Might be none of that, TC currently sends 0

Reply:
It's item field flags. I tested this field by opening a locked container which has a different item flag when opened (the locked bit seems to have a different purpose). also without sending the item flags here the client will bug out so that items in your inventory wont update properly anymore.